### ISSUE TITLE
`vue create .` command crashes silently in debian

### ISSUE THREAD

**Issue by ryanrain** [vuejs/vue-cli#4179]
### Version
3.8.4

### Environment info
```
Environment Info:

  System:
    OS: Linux 4.19 Debian GNU/Linux buster/sid undefined
    CPU: (4) x64 Intel(R) Core(TM) i5-7300HQ CPU @ 2.50GHz
  Binaries:
    Node: 11.10.0 - /usr/bin/node
    Yarn: 1.16.0 - /usr/bin/yarn
    npm: 6.7.0 - /usr/bin/npm
  Browsers:
    Chrome: Not Found
    Firefox: 60.5.0
  npmGlobalPackages:
    @vue/cli: 3.8.4


```

### Steps to reproduce
1. running debian 10, install vue cli with either yarn or npm
2. run `vue create .` from the terminal
3. create in current folder? yes

### What is expected?
to continue with the creation process

### What is actually happening?
the script freezes. arrow navigation to select a standard or custom install does not work. not even ctrl-c exits.

**Comment by mix3d** [comment]
This is crashing on Windows 10 as well

**Comment by snoopdouglas** [comment]
This is happening for me in a fresh `node:dubnium` container.

After the hang, the node process doesn't appear to be using much CPU.

**Comment by MRVDH** [comment]
Porbably the same as #4181

**Comment by sodatea** [comment]
Fixed in inquirer@6.4.1
Now please try to clean your npm/yarn cache and reinstall `@vue/cli`
